qiita_ware/ebi.py

```diff
@@ -272,8 +271,8 @@ def generate_study_xml(self):
 
         Returns
         -------
-        str
-            string with study XML values
+        Element
+            Object with study XML values
```

Comment:
I can do that but AFAIK is not standard (however, this is not a standard type). What about xml.etree.ElementTree.Element?